src/commands/sites/list.js

```diff
@@ -18,8 +18,9 @@ class SitesListCommand extends Command {
         const siteInfo = {
           id: site.id,
           name: site.name,
-          ssl_url: site.ssl_url,
-          account_name: site.account_name
```

Comment:
i need to doublecheck this as i lack context on this specific command but whats up with the rename of these props?

Reply:
`site.ssl_url` is always defined in the API, even if the site is not deployed.

I needed to check if site is deployed, so I used `site.published_deploy`; because published_deploy also contained ssl_url, I decided to use that instead of `site.ssl_url`

I'm not sure if they are the same though. An alternative approach would be to keep using `site.ssl_url` and only use `published_deploy` as a flag.

Reply:
oh man ok. thanks very much. i dont have bandwidth to look at this right now but appreciate the input and will need to come back to this to confirm that these are the right values.

Reply:
Hey, just fyi. There is one more case where cli output is broken — the site is deployed, but the recent build fails. It seems, that the API data doesn't contain any information about the recent broken build.

A possible solution might be to list the deployments for each site, and calculate status from there, but probably it's better to add the status of most recent build to the API.

In any case, this MR doesn't cover all use cases, so feel free to close.

Reply:
thank you. still very tied up here but will get to it eventually. hugely appreciate the PR though!